Frontend/RecRock-Calendar-Frontend/src/app/calendar/components/user-settings/user-settings.component.scss

```diff
@@ -0,0 +1,97 @@
+.container{
+  padding-top: 5%;
+  max-width: 700px;
```

Comment:
in order to make it easy to change the dimensions later, please add a variable called displayUnit in the theming CSS or in the variables CSS and assign to it a basic dimension, for example, 25px. Then you can use it everywhere (for example max-width: $displayUnit * 28, or margin-top: $displayUnit * 0.2).